setup.py

```diff
@@ -70,8 +70,19 @@
         "scipy",
     ],
     extras_require={
-        "cli": ["argcomplete"],
-        'platemotion': ['platemotion', 'astropy'],  # will be removed in the coming new version of plate motion script
+        "cli": ["argcomplete", "rich"],
+        "platemotion": ["platemotion", "astropy"],  # will be removed in the coming new version of plate motion script
+        "gmtsar": ["gdal"],
+        "solidearth": ["pysolid"],
+        "gps": ["proj", "dateutil"],
+        "progress": ["tqdm"],
+        "isce": ["isce"],
+        "legacy": ["pyaps"],
+        "extra": ["gdal", "pillow"],
+        "solvers": ["mosec"],
+        "gbis": ["geoid"],
+        "kite": ["kite"],
+        "fractal": ["pyfftw"],
```

Comment:
With this many extras, it might be worth adding an `all` target that have all the extra requirements in it (except `platemotion` maybe). 

Though, TBF I'll likely never spell out any of the extras as I'll build a conda env first and then pip install into it with `--no-deps`.

Reply:
Good idea!
Done.
I also left out "legacy".